src/components/brew/BrewList.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { BaseBrewingApi } from '../../types/BrewingType';
+import BrewDayInList from './BrewDayInList';
+
+interface BrewListTypeProps {
+  brews: BaseBrewingApi[];
+  callback: (arg: number) => void;
+  current: number;
+}
+
+const BrewList: React.FC<BrewListTypeProps> = ({
+  brews,
+  callback,
+  current,
+}) => {
+  //   take days as the keys
+  const daysOfBrewing = brews.reduce((r, a) => {
+    r[new Date(a.startedAt).toLocaleDateString()] =
```

Comment:
`new Date(a.startedAt).toLocaleDateString()` by som vytiahol do nejakej premennej, lebo to pouzivas tri krat to iste